app/assets/javascripts/discourse/app/components/post/menu.gjs

```diff
@@ -14,7 +14,7 @@ import UserTip from "discourse/components/user-tip";
 import concatClass from "discourse/helpers/concat-class";
 import DAG from "discourse/lib/dag";
 import { applyMutableValueTransformer } from "discourse/lib/transformer";
-import { userPath } from "discourse/lib/url";
+import { smallUserAttrs } from "discourse/lib/user-list-attrs";
```

Comment:
I'd suggest moving this function from `discourse/lib/user-list-attrs` to `discourse/components/small-user-list` because once the Glimmer version is gone, this function won't be used anywhere else